src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -517,6 +567,50 @@ export function GraphWrapper({
 							showing {graphRows.length} item{graphRows.length ? 's' : ''}
 						</span>
 					)}
+					<div className="actioncombo">
+						<button
+							type="button"
+							aria-controls="repo-columnsettings-list"
+							aria-expanded={columnSettingsExpanded}
+							aria-haspopup="listbox"
+							id="repo-actioncombo-label"
+							className="actioncombo__label"
+							role="combobox"
+							aria-activedescendant={
+								columnSettingsExpanded
+									? `repo-actioncombo-item-${reposList.findIndex(
+											item => item.path === currentRepository?.path,
+									  )}`
+									: undefined
```

Comment:
this should be looking against the graph columns

Reply:
Done in commit: `Fixed ids/keys prefix used for the columns combo`